File: applied/2_i_wanna_be_your_lover/answer/high.java
import java.util.*;

public class Main {
    public static void main(String[] args) throws Exception {
        Scanner scan = new Scanner(System.in);
        
        int a = scan.nextInt();
        int b = scan.nextInt();
        int c = scan.nextInt();
        
        double x1 = (-b + Math.sqrt(b*b - 4*a*c)) / (2 * a);
        double x2 = (-b - Math.sqrt(b*b - 4*a*c)) / (2 * a);
        
        System.out.printf("%.1f\n", x1);
        System.out.printf("%.1f\n", x2);

        scan.close();
    }
}

File: applied/02_i_wanna_be_your_lover/answer/high.c
#include <stdio.h>
#include <math.h>
int main(void){
    int a, b, c;
    float x1, x2;

    scanf("%d", &a);
    scanf("%d", &b);
    scanf("%d", &c);

    x1 = (-b + sqrt(b*b - 4*a*c)) / (2 * a);
    x2 = (-b - sqrt(b*b - 4*a*c)) / (2 * a);

    printf("%.1f\n", x1);
    printf("%.1f\n", x2);

    return 0;
}
